src/native-stack/navigators/createNativeStackNavigator.tsx

```diff
@@ -67,13 +66,13 @@ function NativeStackNavigator(props: NativeStackNavigatorProps) {
 
   return (
     <NativeStackView
-      {...rest}
+      {...props}
```

Comment:
why change `rest` to `props`?

Reply:
May just be more of a personal thing but reading `...rest` in isolation can give some indirection where using `props` would explictly tell the reader it is the props passed into this component without having to go back to where the component is declared and then check that `...rest` is in fact spreading component props. Happy to revert this change to keep using rest though.

Reply:
We keep the implementation of this navigator kind of similar to how other react-navigation navigators are implemented, and `rest` is being used there, so I would rather keep it that way.